### ISSUE TITLE
7/10 Task Create SOP & Cheklist for using bookdown.

### ISSUE THREAD

**Issue by anthonycpichardo** [lzim/teampsd#1370]
create an SOP & Checklist for using the RProject and Knitting the .Rmd

a. update the original manual resource using Pull Request

b. re-knit once the update is complete usign the RProject

**Comment by anthonycpichardo** [comment]
duplicate of #1364 closing
